rct229/ruletest_engine/ruletest_jsons/section6/rule_6_7.json

```diff
@@ -0,0 +1,394 @@
+                                                        "floor_area": 92.90303999999999,
+                                                        "occupant_multiplier_schedule": "Occupancy Schedule 1",
+                                                        "interior_lighting": [
+                                                            {
+                                                                "id": "Lighting 1",
+                                                                "daylighting_control_type": "STEPPED",
+                                                                "lighting_multiplier_schedule": "Lighting Schedule",
+                                                                "are_schedules_used_for_modeling_daylighting_control": true
+                                                            }
+                                                        ]
+                                                    }
+                                                ],
+                                                "surfaces": [
+                                                    {
+                                                        "id": "surface_1",
+                                                        "adjacent_to": "EXTERIOR",
+                                                        "subsurfaces": [
+                                                            {
+                                                                "id": "window_1",
+                                                                "classification": "WINDOW"
+                                                            }
+                                                        ]
+                                                    },
+                                                    {
+                                                        "id": "surface_2",
+                                                        "adjacent_to": "EXTERIOR",
+                                                        "subsurfaces": [
+                                                            {
+                                                                "id": "window_2"
+                                                            }
+                                                        ]
+                                                    }
+                                                ]
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
+    },
+    "rule-6-7-c": {
+        "Section": 6,
+        "Rule": 7,
+        "Test": "c",
+        "test_description": "Proposed building is modeled with daylighting controls directly or through schedule adjustments.",
```

Comment:
Here the description says there's daylighting but it appears the JSON does not have any

Reply:
Fixed. Could you verify the new version of the updated 6-7 JSON file?